File: contests/abc144/abc144_b.py
# B - 81
# https://atcoder.jp/contests/abc144/tasks/abc144_b

n = int(input())
for i in range(1, 10):
    for j in range(1, 10):
        if n == i * j:
            print('Yes')
            exit()

print('No')

File: contests/abc144/abc144_b.cpp
// B - 81
// https://atcoder.jp/contests/abc144/tasks/abc144_b

#include <iostream>
using namespace std;

int main() {
    int n; cin >> n;
    bool ans = false;

    for(int i=1;i<=9;i++) {
        for(int j=1;j<=9;j++) {
            if (i*j == n) ans = true;
        }
    }
    cout << ((ans) ? "Yes" : "No") << endl;
}
